app/serializers/framework_assessment_serializer.rb

```diff
@@ -9,7 +9,9 @@ class FrameworkAssessmentSerializer
   has_many_if_included :responses, serializer: FrameworkResponseSerializer, &:framework_responses
   has_many_if_included :flags, serializer: FrameworkFlagSerializer, &:framework_flags
 
-  attributes :confirmed_at, :created_at, :nomis_sync_status
+  attributes :completed_at, :confirmed_at, :created_at, :nomis_sync_status
+
+  attribute :amended_at, if: proc { |object| object.respond_to?(:amended_at) }
```

Comment:
instead of here, can we put this in the dedicated PER and PERs serialisers to avoid the responds to? https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/serializers/person_escort_record_serializer.rb and https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/serializers/person_escort_records_serializer.rb
Also could you add the `completed_at` to the PERs serialiser as it doesn't inherit from the framework assessment serialiser

Reply:
Good shout - done 👍